generators/app/index.js

```diff
@@ -86,6 +86,13 @@ module.exports = fountain.Base.extend({
             'isparta-loader': '^2.0.0'
           }
         });
+        if (this.options.js === 'babel') {
+          _.merge(pkg, {
+            devDependencies: {
+              'gulp-env': '^0.4.0'
```

Comment:
This should be in [generator-fountain-gulp](https://github.com/FountainJS/generator-fountain-gulp/blob/7b482f11b3c1bcd4c68c17bb5f46e1511f538823/generators/app/index.js) or i missed something